src/extension/ios/iOSPlatform.ts

```diff
@@ -35,6 +39,20 @@ export class IOSPlatform extends GeneralMobilePlatform {
 
     private static RUN_IOS_SUCCESS_PATTERNS = ["BUILD SUCCEEDED"];
 
+    public static showDevMenu(deviceId?: string): Q.Promise<void> {
+        if (!this.remoteExtension) {
+            this.remoteExtension = RemoteExtension.atProjectRootPath(SettingsHelper.getReactNativeProjectRoot());
+        }
+        return this.remoteExtension.showDevMenu(deviceId);
+    }
+
+    public static reloadApp(deviceId?: string): Q.Promise<void> {
+        if (!this.remoteExtension) {
```

Comment:
Nit: instead of repeating this lazy initialization twice maybe just create a private getter for `this.remoteExtension`?